package.json

```diff
@@ -30,6 +30,11 @@
     "canjs-plugin",
     "donejs"
   ],
+  "steal": {
+    "plugins": [
+      "steal-css"
```

Comment:
It's a steal bug: https://github.com/stealjs/steal/pull/963

Even though you're getting that 404, it should still load.